automation/deployment/deployer.py

```diff
@@ -752,28 +754,29 @@ def run_command(
     live: bool = True,
     log_file_handler: typing.Optional[typing.IO[str]] = None,
 ) -> (str, str, int):
-    if workdir:
-        command = f"cd {workdir}; " + command
     if args:
-        command += " " + " ".join(args)
-
-    process = subprocess.Popen(
-        command,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        stdin=subprocess.PIPE,
-        shell=True,
-    )
+        command = [command] + args
```

Comment:
While I see that `command` is only passed as a one-word string, perhaps best to verify it by splitting on space and taking the first item.